nixos/modules/services/networking/prosody.nix

```diff
@@ -839,9 +860,8 @@ in
     users.users.prosody = mkIf (cfg.user == "prosody") {
       uid = config.ids.uids.prosody;
       description = "Prosody user";
-      createHome = true;
       inherit (cfg) group;
-      home = "${cfg.dataDir}";
+      home = cfg.dataDir;
```

Comment:
is this still necessary for prosody to function?

Reply:
I had the same question. Unsure, though.

Reply:
better to leave it then. :+1: